profile/2020-09-01-hybrid.json

```diff
@@ -0,0 +1,625 @@
+        },
+        {
+          "resourceType": "providers",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/resources/resource-manager/Microsoft.Resources/stable/2018-05-01/resources.json"
+        },
+        {
+          "resourceType": "resourceGroups",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/resources/resource-manager/Microsoft.Resources/stable/2018-05-01/resources.json"
+        },
+        {
+          "resourceType": "resources",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/resources/resource-manager/Microsoft.Resources/stable/2018-05-01/resources.json"
+        },
+        {
+          "resourceType": "subscriptions/operationresults",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/resources/resource-manager/Microsoft.Resources/stable/2018-05-01/resources.json"
+        },
+        {
+          "resourceType": "subscriptions/providers",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/resources/resource-manager/Microsoft.Resources/stable/2018-05-01/resources.json"
+        },
+        {
+          "resourceType": "subscriptions/resourceGroups",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/resources/resource-manager/Microsoft.Resources/stable/2018-05-01/resources.json"
+        },
+        {
+          "resourceType": "subscriptions/resourceGroups/resources",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/resources/resource-manager/Microsoft.Resources/stable/2018-05-01/resources.json"
+        },
+        {
+          "resourceType": "subscriptions/resources",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/resources/resource-manager/Microsoft.Resources/stable/2018-05-01/resources.json"
+        },
+        {
+          "resourceType": "subscriptions/tagNames",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/resources/resource-manager/Microsoft.Resources/stable/2018-05-01/resources.json"
+        },
+        {
+          "resourceType": "subscriptions/tagNames/tagValues",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/resources/resource-manager/Microsoft.Resources/stable/2018-05-01/resources.json"
+        }
+      ]
+    },
+    "microsoft.storage": {
+      "2019-07-07": [
+        {
+          "resourceType": "checkNameAvailability",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/storage/resource-manager/Microsoft.Storage/stable/2017-10-01/storage.json"
```

Comment:
Though the tag is "2019-07-07", but actually the swagger file is 2017-10-01:
https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/storage/resource-manager/Microsoft.Storage/stable/2017-10-01/storage.json

Reply:
The correct version here is 2019-06-01